src/main/java/mekanism/common/integration/multipart/MultipartTileNetworkJoiner.java

```diff
@@ -0,0 +1,83 @@
+package mekanism.common.integration.multipart;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import io.netty.buffer.ByteBuf;
+import mcmultipart.api.container.IMultipartContainer;
+import mcmultipart.api.slot.EnumFaceSlot;
+import mcmultipart.api.slot.IPartSlot;
+import mcmultipart.api.world.IMultipartBlockAccess;
+import mekanism.common.base.ITileNetwork;
+import net.minecraft.tileentity.TileEntity;
+import net.minecraft.world.IBlockAccess;
+
+public class MultipartTileNetworkJoiner implements ITileNetwork
```

Comment:
sure, byte wise it's not much of a difference, but at the absolute least it needs a comment in each file about why its reading a byte and doing nothing with it - so it would look even tidier if you can manage to put it in the Cap. How exactly you go about that is mostly up to you.

How many other places bypass the cap to send messages?